client/src/css/stylesheet.scss

```diff
@@ -35,177 +42,315 @@ body {
       display: flex;
       align-items: center;
 
+      #status-bar {
+        width: auto;
+        height: 100%;
+        text-align: center;
+        flex-grow: 1;
+
+        #status {
+          position: relative;
+          top: 50%;
+          transform: translateY(-50%);
+          margin: 0;
+          font-size: 2em;
+        }
+      }
+    }
+
+    #left-flex {
+      position: absolute;
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: center;
+      align-content: space-between;
+      align-items: space-between;
+      left: 0%;
+      top: 12%;
+      width: 50%;
+      height: 88%;
+      overflow: visible;
+
       #left-arrow,
       #right-arrow {
-        width: 7%;
+        width: 5%;
       }
 
       #left-arrow {
-        height: 100%;
-        text-align: center;
+        position: relative;
+        height: 20%;
+        top: 0%;
 
         #left-icon {
           position: absolute;
-          top: 50%;
-          bottom: 50%;
-          height: 75%;
-          transform: translate(-50%, -50%);
+          left: 200%;
+          top: 0%;
+          height: 55%;
+        }
+      }
+
+      #speedometer-wrapper {
+        position: relative;
+        width: 90%;
+        height: 50%;
+
+        #speedometer {
+          position: relative;
+          height: 190%;
+          width: 100%;
+          overflow: visible;
+
+          #speedometer-path {
+            fill-opacity: 0;
+            stroke-linecap: "butt";
+            stroke: #ccc;
+            stroke-opacity: 1;
+            stroke-width: 2rem;
+          }
+
+          #speedometer-text {
+            text-anchor: middle;
+            alignment-baseline: middle;
+            fill: white;
+            font-family: "Open Sans";
+            font-size: 15em;
```

Comment:
Intentional for now. Most of this is only partially scalable currently. Since we know the dimensions we will use it on I'm not amazingly concerned with supporting a bunch of screen sizes unless you want to?